fdbserver/Resolver.actor.cpp

```diff
@@ -88,8 +88,7 @@ class RecentStateTransactionsInfo {
 		// sends this request has them and will apply them via applyMetadataToCommittedTransactions();
 		// and other proxies will get this version's state txns as a prior version.
 		for (; stateTransactionItr != endItr; ++stateTransactionItr) {
-			shardChangedOrStateTxn =
-			    shardChangedOrStateTxn || stateTransactionItr->value.first || stateTransactionItr->value.second.size();
+			shardChangedOrStateTxn = shardChangedOrStateTxn || stateTransactionItr->value.first;
```

Comment:
does this line change fix a bug?

maybe when `shardChangedOrStateTxn` is false, we can just `continue`?

Reply:
It makes it more efficient, we do not need to send to all tLogs if a different CP modified the transaction state store.